src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_operation_factory.py

```diff
@@ -0,0 +1,365 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+## SDK COMMAND OPERATIONS
```

Comment:
This seems like a lot of tedious boilerplate code. Could we not specify the operation by a string and then import it lazily within the core execution rather than with a callable for each command?